### ISSUE TITLE
Poor reproduction results

### ISSUE THREAD

**Issue by liangjxiong** [Jeff-Zilence/TransGeo2022#40]
Hello! I am very interested in your work and it has greatly inspired me. This is an outstanding job. I encountered difficulties in reproducing the results of my paper and urgently need your help. The experimental results of the paper are presented in the VIGOR dataset: Percentage top1:61.48, top5:87.54, top10:91.88, and top1%: 99.56. The results I validated using the trained model you provided are: Percentage top1: 57.56677121946583, top5: 83.97870924817032, top10: 89.24246744606026, top1%: 99.41070240471439, time: 5480.254483938217. I haven't changed any parameters, R @ 1 has a significant difference, what should I do? Hope for your reply! Thank you!

**Comment by ssfkfffddj** [comment]
Hello, what is the accuracy of the first stage of your training?

**Comment by liangjxiong** [comment]
Hello! I didn't train. I am only using the weights you provided to validate the model.

**Comment by ssfkfffddj** [comment]
I'm not author.

**Comment by ssfkfffddj** [comment]
Thanks for your reply.And I'm not the  Author.

**Comment by Jeff-Zilence** [comment]
Not sure what is going on here. I will need more details of your evaluation code to figure out. Alternatively, you can run the [run_VIGOR.sh](https://github.com/Jeff-Zilence/TransGeo2022/blob/main/run_VIGOR.sh) to reproduce the results. Please do not change the hyperparameters including the batch size.
